dist/common/scripts/scylla_util.py

```diff
@@ -87,12 +87,16 @@ def datadir():
 def scyllabindir():
     return str(scyllabindir_p())
 
-def curl(url, byte=False):
+# @param headers semicolon ; separated values of k:v
```

Comment:
Please change to a dict and convert in the function.

Reply:
I didn't see this fixed.

Reply:
fixed in https://github.com/scylladb/scylla/pull/6632/commits/97c250e4d19b11d7ca1ed2937cc64cdcebda4441